src/applications/disability-benefits/all-claims/pages/addDisabilities.js

```diff
@@ -95,20 +95,16 @@ export const uiSchema = {
       'ui:description': newOnlyAlert,
       'ui:options': {
         hideIf: formData =>
-          !newConditionsOnly(formData) || hasClaimedConditions(formData),
+          !(newConditionsOnly(formData) && !claimingNew(formData)),
       },
     },
+    // Only show this alert if the veteran is claiming both rated and new
+    // conditions but no rated conditions were selected
     'view:increaseAndNewAlert': {
       'ui:description': increaseAndNewAlert,
       'ui:options': {
-        hideIf: formData => {
-          // Only show this alert if the veteran is claiming both rated and new
-          // conditions but no rated conditions were selected
-          return (
-            !claimingRated(formData) &&
-            (!newAndIncrease(formData) || hasClaimedConditions(formData))
-          );
-        },
+        hideIf: formData =>
+          !(newAndIncrease(formData) && !hasClaimedConditions(formData)),
```

Comment:
@Mottie This page was confusing to figure out what the purpose of each alert message was, but from what I understand, both error messages serve the same purpose, we just have different wording depending on if the user claimed increase or not. The logic here is mostly the same, I just dropped some redundant checks and slightly adjusted the operators to make it more clear what is being checked.

Reply:
Hmm, I was just testing this... when I do the following:

- Don't select a rated disability, click "Continue"
- The new condition page will show "Unknown condition" (we need to fix this), click "Continue"
- You'll see the alert about adding a disability
- Go back to the rated disability page and add one
- "Continue" x2
- No alert is shown and you can't get past the new disabilities page

I used this data starting from the `/rated-disabilities` page

<details><summary>SiP Data</summary>

<!-- leave a blank line above -->
```json
{
  "ratedDisabilities": [
    {
      "name": "Diabetes mellitus0",
      "ratedDisabilityId": "0",
      "ratingDecisionId": "63655",
      "diagnosticCode": 5238,
      "decisionCode": "SVCCONNCTED",
      "decisionText": "Service Connected",
      "ratingPercentage": 100,
      "disabilityActionType": "NONE",
      "view:selected": false
    },
    {
      "name": "Diabetes mellitus1",
      "ratedDisabilityId": "1",
      "ratingDecisionId": "63655",
      "diagnosticCode": 5238,
      "decisionCode": "SVCCONNCTED",
      "decisionText": "Service Connected",
      "ratingPercentage": 100,
      "disabilityActionType": "NONE"
    }
  ],
  "view:disabilitiesClarification": {},
  "newDisabilities": [
    {
      "view:descriptionInfo": {}
    }
  ],
  "view:newDisabilityErrors": {},
  "serviceInformation": {
    "reservesNationalGuardService": {
      "view:isTitle10Activated": false,
      "obligationTermOfServiceDateRange": {
        "from": "2007-05-22",
        "to": "2008-06-05"
      },
      "unitName": "Reserves"
    },
    "view:separationLocation": {},
    "separationLocation": {
      "id": "98339",
      "label": "Fort Dix",
      "widget": "autosuggest"
    },
    "servicePeriods": [
      {
        "serviceBranch": "Air Force Reserve",
        "dateRange": {
          "from": "2001-03-21",
          "to": "2014-07-21"
        }
      },
      {
        "serviceBranch": "Air Force",
        "dateRange": {
          "from": "2014-07-31",
          "to": "2021-03-01"
        }
      }
    ],
    "view:militaryHistoryNote": {}
  },
  "view:claimType": {
    "view:claimingNew": true,
    "view:claimingIncrease": true
  },
  "servedInCombatZonePost911": false,
  "view:selectablePtsdTypes": {},
  "view:ptsdTypeHelp": {},
  "view:upload781ChoiceHelp": {},
  "incident0": {
    "unitAssignedDates": {},
    "incidentLocation": {
      "country": "USA"
    }
  },
  "incident1": {
    "unitAssignedDates": {},
    "incidentLocation": {
      "country": "USA"
    }
  },
  "incident2": {
    "unitAssignedDates": {},
    "incidentLocation": {
      "country": "USA"
    }
  },
  "view:upload781aChoiceHelp": {},
  "secondaryIncident0": {
    "unitAssignedDates": {},
    "incidentLocation": {
      "country": "USA"
    },
    "otherSourcesHelp": {}
  },
  "view:otherSourcesHelp": {},
  "secondaryIncident1": {
    "unitAssignedDates": {},
    "incidentLocation": {
      "country": "USA"
    },
    "otherSourcesHelp": {}
  },
  "secondaryIncident2": {
    "unitAssignedDates": {},
    "incidentLocation": {
      "country": "USA"
    },
    "otherSourcesHelp": {}
  },
  "physicalChanges": {},
  "mentalChanges": {},
  "workBehaviorChanges": {},
  "socialBehaviorChanges": {},
  "view:ancillaryFormsWizardIntro": {},
  "view:unemployabilityHelp": {},
  "unemployability": {
    "view:recordsInfo": {},
    "view:unemployabilityDatesDesc": {},
    "view:grossIncomeAdditionalInfo": {},
    "view:supplementalBenefitsHelp": {},
    "view:substantiallyGainfulEmploymentInfo": {},
    "view:statementsAreTrue": {},
    "view:informOfReturnToWork": {}
  },
  "view:privateMedicalFacility": {},
  "view:upload4192Choice": {},
  "view:downloadInfo": {},
  "view:uploadServiceTreatmentRecordsQualifier": {},
  "view:hasEvidence": true,
  "view:hasEvidenceFollowUp": {
    "view:selectableEvidenceTypes": {},
    "view:evidenceTypeHelp": {}
  },
  "view:vaMedicalRecordsIntro": {},
  "view:uploadPrivateRecordsQualifier": {
    "view:aboutPrivateMedicalRecords": {},
    "view:privateRecordsChoiceHelp": {}
  },
  "view:privateRecordsChoiceHelp": {},
  "view:faqAccordion": {},
  "phoneAndEmail": {
    "primaryPhone": "4445551212",
    "emailAddress": "test2@test1.net"
  },
  "mailingAddress": {
    "view:livesOnMilitaryBaseInfo": {},
    "country": "USA",
    "addressLine1": "123 MAIN ST",
    "addressLine2": "BEN FRANKLIN VILLAGE",
    "city": "APO",
    "state": "AE",
    "zipCode": "09028"
  },
  "view:contactInfoDescription": {},
  "view:bankAccount": {
    "view:hasPrefilledBank": true
  },
  "view:terminallyIllInfo": {},
  "view:waiveRetirementPayDescription": {},
  "standardClaim": false,
  "view:fdcWarning": {},
  "view:originalBankAccount": {
    "view:bankAccountType": "Checking",
    "view:bankAccountNumber": "*********1234",
    "view:bankRoutingNumber": "*****2115",
    "view:bankName": "Comerica"
  }
}
```
</details>

Reply:
Added validation to the rated disabilities page to require one to be selected. The issue with the new disabilities page will be handled in a separate ticket.